scripts/globals/garrison_data.lua

```diff
@@ -0,0 +1,622 @@
+        mobBoss = "Hunting_Chief",
+        xPos = -247, -- needs adjusting
+        yPos = -19,
+        zPos = 310,
+        xChange = -2,
+        zChange = 0,
+        xSecondLine = 0,
+        zSecondLine = -2,
+        xThirdLine = 0,
+        zThirdLine = -4,
+        rot = 0,
+    },
+    [xi.zone.BEAUCEDINE_GLACIER] =
+    {
+        itemReq = xi.items.TIGER_LEATHER_MISSIVE,
+        textRegion = 8,
+        levelCap = 40,
+        mobBoss = "Gigas_Overseer",
+        xPos = -25, -- needs adjusting
+        yPos = -60,
+        zPos = -110,
+        xChange = -2,
+        zChange = -1,
+        xSecondLine = 0,
+        zSecondLine = -1,
+        xThirdLine = 0,
+        zThirdLine = -2,
+        rot = 220,
+    },
+    [xi.zone.THE_SANCTUARY_OF_ZITAH] =
+    {
+        itemReq = xi.items.HOUND_FANG_SACK,
+        textRegion = 11,
+        levelCap = 40,
+        mobBoss = "Goblin_Doyen",
+        xPos = -43,
+        yPos = 1,
+        zPos = -140,
+        xChange = -2,
+        zChange = 0,
+        xSecondLine = 0,
+        zSecondLine = 2,
+        xThirdLine = 0,
+        zThirdLine = 4,
+        rot = 180,
+    },
+    [xi.zone.YUHTUNGA_JUNGLE] =
+    {
+        itemReq = xi.items.SHEEP_LEATHER_MISSIVE,
+        textRegion = 14,
+        levelCap = 40,
+        mobBoss = "Sahagin_Patriarch",
+        xPos = -248,
+        yPos = 1,
+        zPos = -392,
+        xChange = -2,
+        zChange = 0,
+        xSecondLine = 0,
+        zSecondLine = 2,
+        xThirdLine = 0,
+        zThirdLine = 4,
+        rot = 180,
+    },
+    [xi.zone.XARCABARD] =
+    {
+        itemReq = xi.items.BEHEMOTH_LEATHER_MISSIVE,
+        textRegion = 9,
+        levelCap = 50,
+        mobBoss = "Demon_Aristocrat",
+        xPos = 216, -- needs adjusting
+        yPos = -22,
+        zPos = -208,
+        xChange = 2,
+        zChange = 0,
+        xSecondLine = 0,
+        zSecondLine = 2,
+        xThirdLine = 0,
+        zThirdLine = 4,
+        rot = 90,
+    },
+    [xi.zone.EASTERN_ALTEPA_DESERT] =
+    {
+        itemReq = xi.items.DHALMEL_LEATHER_MISSIVE,
+        textRegion = 12,
+        levelCap = 50,
+        mobBoss = "Centurio_XIII-V",
+        xPos = -245,
+        yPos = -9,
+        zPos = -249,
+        xChange = 0,
+        zChange = 2,
+        xSecondLine = 2,
+        zSecondLine = 0,
+        xThirdLine = 4,
+        zThirdLine = 0,
+        rot = 0,
+    },
+    [xi.zone.YHOATOR_JUNGLE] =
+    {
+        itemReq = xi.items.COEURL_LEATHER_MISSIVE,
+        textRegion = 15,
+        levelCap = 50,
+        mobBoss = "Tonberry_Decimator",
+        xPos = 214,
+        yPos = 1,
+        zPos = -80,
+        xChange = 1,
+        zChange = 2,
+        xSecondLine = 2,
+        zSecondLine = -1,
+        xThirdLine = 4,
+        zThirdLine = -2,
+        rot = 0,
+    },
+    [xi.zone.CAPE_TERIGGAN] =
+    {
+        itemReq = xi.items.BUNNY_FANG_SACK,
+        textRegion = 13,
+        levelCap = 75, -- 75 Mobs/NPCs (Uncapped)
```

Comment:
Does this comment need to be here?

Reply:
Changed level cap to 99 and pull from `xi.settings.main.MAX_LEVEL` in this case